telemetry/telegraf/convert.go

```diff
@@ -58,28 +56,55 @@ func (c *Converter) Convert(body []byte) ([]telemetry.FrameWrapper, error) {
 	return c.convertWithLabelsColumn(metrics)
 }
 
+func getHash(m influx.Metric) uint32 {
+	h := fnv.New32a()
+	_, _ = h.Write([]byte(m.Name()))
```

Comment:
do we also need to list the fields?

Reply:
Just checked how `HashID` calculated in previous Telegraf library:

```go
func (m *metric) HashID() uint64 {
	h := fnv.New64a()
	h.Write([]byte(m.name))
	h.Write([]byte("\n"))
	for _, tag := range m.tags {
		h.Write([]byte(tag.Key))
		h.Write([]byte("\n"))
		h.Write([]byte(tag.Value))
		h.Write([]byte("\n"))
	}
	return h.Sum64()
}
```

So looks like it's fine without fields. Also made sure that Tags sorted here (the parser library sorts them on Metric create).